src/providers/exploreState.tsx

```diff
@@ -461,10 +465,14 @@ function exploreReducer(
       if (payload === state.tabValue) {
         return state;
       }
+      const { entityPageState } = state;
+      const entityState = entityPageState[payload];
       return {
         ...state,
-        filterCount: state.entityPageState[payload].filterCount,
-        filterState: state.entityPageState[payload].filterState,
+        categoryGroupConfigs: entityState.categoryGroupConfigs,
```

Comment:
`SelectEntityType` action should update `entityPageState` values to top level state to keep values in sync.